docs/UserGuide.md

```diff
@@ -490,31 +509,34 @@ Some Frequently Asked Questions:
 ---------------------------------------------------------------------------------------------------------------------
 
 ## Glossary
+1. **CS1101S:** CS1101S Programming Methodology is a Computer Science module in National Univeristy of Singapore (NUS), School of Computing. This module introduces the concepts of programming and computational problem-solving, and is the first and foremost introductory module to computing, hence it is usually taken by first year Computer Science undergraduate students.
+
+2. **Teaching Assistant:** Teaching assistants (TA) are undergraduate students who help and support teachers or lecturers to provide and coordinate effective classroom instruction. For CS1101S, the role of a TA involves planning, preparing and delivering weekly tutorial sessions, marking assignments and monitoring their students' progress and engagement levels.
```

Comment:
TAs can also be grad students right

Reply:
Updated to "people" instead of "undergraduate students". Also mentioned that specifically CS1101S TAs are undergraduate students.